courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/1_core_tensorflow.ipynb

```diff
@@ -106,13 +106,21 @@
     "x = tf.Variable(2.0, dtype=tf.float32, name='my_variable')"
```

Comment:
<p>A lot of lab task #1s haha. Maybe break into more tasks so we don't do lab task #1 a lot of times. Or maybe header type. None of this really matters though lol. All personal preference for style. These labs are great as is!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/704/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>I may just leave as it to keep simple. </p>